CODEOWNERS

```diff
@@ -1,2 +0,0 @@
-# Comment line immediately above ownership line is reserved for related gus information. Please be careful while editing.
```

Comment:
Should a `CODEOWNERS` nested under the `.github` directory, that includes the language team alias be added?

Reply:
Fixed